### PR TITLE
Add dockerfile for java on CentOS

### PR DESCRIPTION
**Pull request by kahootali**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM stakater/centos-base-image:7
+
+LABEL name="Stakater Java image on CentOS" \    
+      maintainer="Stakater <stakater@aurorasolutions.io>" \
+      vendor="Stakater" \
+      release="1" \
+      summary="A Java image based on CentOS" 
+
+USER root
```
Comment (rasheedamir) [inline Dockerfile]: add reason why you need to become root first

File: README.md [hunk 2]
```diff
@@ -1 +1,10 @@
-# dockerfile-java-centos
\ No newline at end of file
+# dockerfile-java-centos
+
+This is a java image based on CentOS which is used by Stakater for its apps. The base image used is [stakater/centos-base-image](https://github.com/stakater/dockerfile-base-centos). 
+
+You can run this image by:
+`docker run -it stakater/java-centos-image:1.8 /bin/bash`
```
Comment (rasheedamir) [inline README.md]: The docker image name should be just `java-centos`

**Comment by stakater-user** [comment]
@kahootali Image is available for testing. `docker pull docker.io/stakater/java-centos:7-1.8`

**Comment by stakater-user** [comment]
@kahootali Image is available for testing. `docker pull docker.io/stakater/java-centos:7-1.8`

**Review by rasheedamir — APPROVED** [review]

**Pull request merged**